rpcd/playbooks/roles/rpc_maas/tasks/container-storage-checks.yml

```diff
@@ -20,4 +20,3 @@
     owner: "root"
     group: "root"
     mode: "0644"
-  delegate_to: "{{ physical_host }}"
```

Comment:
I'm fairly certain that all check configs for MaaS need to go onto the host. Are you sure this change is necessary?